src/app/dashboard/whitelist/actions.js

```diff
@@ -0,0 +1,273 @@
+//@flow
+
+import * as ui from 'polymath-ui'
+import { TransferManager, SecurityToken } from 'polymathjs'
+import type { Investor } from 'polymathjs/types'
+import { formName as userFormName } from './components/userForm'
+import { formName as editInvestorsFormName } from './components/editInvestorsForm'
+import type { GetState } from '../../../redux/reducer'
+import type { ExtractReturn } from '../../../redux/helpers'
+
+//ac_ = actionCreator
+export const TRANSFER_MANAGER = 'dashboard/whitelist/TRANSFER_MANAGER'
+export const ac_transferManager = (transferManager: TransferManager) => ({ type: TRANSFER_MANAGER, transferManager: transferManager })
+
+export const UPLOAD_CSV = 'dashboard/whitelist/UPLOAD_CSV'
+export const ac_csvUpload = (csvMessage: string, addresses: Array<string>, sell: Array<string>, buy: Array<string>, previewCSVShowing: boolean, ) => ({ type: UPLOAD_CSV, csvMessage, addresses, sell, buy, previewCSVShowing })
+export const UPLOAD_CSV_FAILED = 'dashboard/whitelist/UPLOAD_CSV_FAILED'
+
+export const GET_WHITELIST = 'dashboard/whitelist/GET_WHITELIST'
+export const ac_getWhitelist = (investors: Array<Investor>) => ({ type: GET_WHITELIST, investors })
```

Comment:
what ac_ prefix means here?

Reply:
https://github.com/PolymathNetwork/polymath-issuer/blob/e23bb1d7133d6d4a74ec60fc0864e974f64117ea/src/app/dashboard/whitelist/actions.js#L11

on line 11 I state ac_ is for actionCreator. This is the naming convention I could come up with to make it more clear as there were too many things named transferManager, uploadCsv, etc. 

Let me know if there is a better naming convention and I will update it @bshevchenko